polkadot/runtime/rococo/src/impls.rs

```diff
@@ -42,6 +43,17 @@ enum IdentityMigratorCalls<AccountId: Encode> {
 	PokeDeposit(AccountId),
 }
 
+/// Balance unreserve weight
+pub struct BalanceUnreserveWeight<T>(PhantomData<T>);
+
+impl<T: pallet_balances::Config> Get<Weight> for BalanceUnreserveWeight<T> {
+	fn get() -> Weight {
+		use pallet_balances::WeightInfo;
+
+		T::WeightInfo::force_unreserve()
+	}
+}
+
```

Comment:
can be 

```
pub struct BalanceUnreserveWeight
impl Get<Weight> for BalanceUnreserveWeight{
	fn get() -> Weight {
        weights::pallet_balances::WeightInfo<Runtime>::force_unreserve()
	}
}
```

or maybe even 

```
parameter_types! {
    pub BalanceUnreserveWeight: Weight = weights::pallet_balances::WeightInfo<Runtime>::force_unreserve();
}
```

also please double check that westend and rococo really have something to clean up, otherwise we do not need to include it